content/docs/command-reference/checkout.md

```diff
@@ -33,15 +33,16 @@ for more details.
 
 The execution of `dvc checkout` does the following:
 
-- Scans the `.dvc` and `dvc.lock` files to compare against the data files or
-  directories in the <abbr>workspace</abbr>. DVC knows which data
-  (<abbr>outputs</abbr>) match because the corresponding hash values are saved
-  in the `outs` fields in those files. Scanning is limited to the given
-  `targets` (if any). See also options `--with-deps` and `--recursive` below.
+- Scans all `dvc.lock` and `.dvc` files to compare the hash values of its
+  <abbr>outputs</abbr> against the actual data files or directories in the
+  workspace (similar to `dvc status`). Scanning is limited to any given
```

Comment:
Scanning is limited to a file inside a directory. `.dvc` files inside directories? Very confusing still :(

Reply:
This is meant to contract:

> Scanning is limited to any given targets.
> Files inside directories-tracked-as-a-whole are supported (as targets).

We wanted to summarize the note as much as possible according to previous reviews, and to put it near where the `targets` argument is explained. This is the only place where `targets` is kind of explained in this ref. Adding a paragraph to explain `targets` seemed out of scope, so I did this instead. Should I add a paragraph explaining targets before the bullets instead, so it's easier to explain this concept? 🧠

Reply:
Will follow up in https://github.com/iterative/dvc.org/pull/1384#pullrequestreview-450327267 just below.